examples/dit/README.md

```diff
@@ -119,13 +119,25 @@ Then run `python sample.py -c config-file-path`.
 
```

Comment:
I suggest to remove the section of `finetuning DiT model on a toy dataset` because they are not so useful.